mapviz/src/mapviz.cpp

```diff
@@ -1441,7 +1458,7 @@ void Mapviz::Screenshot()
   }
   else
   {
-    ROS_ERROR("Failed to take screenshot.");
+    ROS_ERROR("Failed to take/opt/ros/kinetic/lib/librqt_mapviz.so screenshot.");
```

Comment:
This looks accidental, did this just get accidentally pasted here?
```suggestion
    ROS_ERROR("Failed to take screenshot.");
```

Reply:
Yeah this was accidental.  Oops